docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,216 @@ Postprocessing
 
 .. currentmodule:: fairlearn.postprocessing
 
+Fairlearn currently supports one postprocessing technique that is based on the
+paper *Equality of Opportunity in Supervised Learning* [#3]_. Unlike other
+mitigation techniques :code:`ThresholdOptimizer` is built to satisfy the
+specified fairness criteria exactly and with no remaining disparity.
+In many cases this comes at the expense of performance, for example, with
+significantly lower accuracy. Regardless, it is a useful data point to compare
+results with.
+
+:code:`ThresholdOptimizer` expects an estimator that provides it with scores.
```

Comment:
Why not `:class:`.ThresholdOptimizer` so that you get the link? In other places too.